README.md

```diff
@@ -0,0 +1,70 @@
+# Static artifact storage
+
+[![License](https://img.shields.io/badge/license-Apache-green.svg?style=flat)](https://raw.githubusercontent.com/lean-delivery/tf-module-static-artifact-storage/master/LICENSE)
+[![Build Status](https://travis-ci.org/lean-delivery/tf-module-static-artifact-storage.svg?branch=master)](https://travis-ci.org/lean-delivery/tf-module-static-artifact-storage)
+
+## Description
+
+Terraform project to setup static artifact storage on AWS
+
+## Usage
+
+### Conditional creation
+
+```
+module "static-artifact-storage" {
+    source = "github.com/lean-delivery/tf-module-static-artifact-storage"
+    
+    region          = "us-west-2"
```

Comment:
What is the purpose of `region` variable here?

Reply:
There is no purpose, its mistake, fixed